tools/data_scripts/fetch_aws_datasets.sh

```diff
@@ -11,19 +11,28 @@ echo "$ROOTDIR"
 if [ -z $1 ]
 then
 	AGENT="craftassist"
+	echo "Defaulting to default agent: '$AGENT'"
 else
 	AGENT=$1
 fi
-CHKSUM=$2
 
-echo "Checksum" $CHKSUM
+if [ -z $2 ]
+then
+	CHECKSUM_FILE="tools/data_scripts/default_checksums/datasets.txt"
```

Comment:
use absolute paths so that this works from any directory - `${ROOTDIR}/tools/data_scripts/default_checksums/datasets.txt`.

Reply:
Fixed in a followup commit.